Building DAG of jobs...
Using shell: /bin/bash
Provided cores: 1 (use --cores to define parallelism)
Rules claiming more threads will be scaled down.
Job counts:
	count	jobs
	1	clean_indelible
	1	concat_fasta
	1	concat_indelible
	1	create_control_files
	1	get_indelible_dict
	1	run_grasp
	1	run_indelible
	1	summarise_indels
	8
Select jobs to execute...

[Tue Mar  2 12:13:12 2021]
rule create_control_files:
    output: indelible/10_control.txt
    jobid: 4
    wildcards: taxon=10

[Tue Mar  2 12:13:13 2021]
Finished job 4.
1 of 8 steps (12%) done
Select jobs to execute...

[Tue Mar  2 12:13:13 2021]
rule run_indelible:
    input: indelible/10_control.txt
    output: indelible_output/10/trees/38.nwk, indelible_output/10/10_TRUE_38.fasta, indelible_output/10/10_ANCESTRAL_38.fasta, indelible_output/10/N0_trees/38.nwk
    jobid: 3
    wildcards: taxon=10, rep=38

[Tue Mar  2 12:13:16 2021]
Finished job 3.
2 of 8 steps (25%) done
Select jobs to execute...

[Tue Mar  2 12:13:16 2021]
rule concat_indelible:
    input: indelible_output/10/10_TRUE_38.fasta, indelible_output/10/10_ANCESTRAL_38.fasta
    output: indelible_output/10/concatenated/10_38.fasta
    jobid: 5
    wildcards: taxon=10, rep=38

[Tue Mar  2 12:13:16 2021]
Finished job 5.
3 of 8 steps (38%) done
Select jobs to execute...

[Tue Mar  2 12:13:16 2021]
rule clean_indelible:
    input: indelible_output/10/N0_trees/38.nwk, indelible_output/10/concatenated/10_38.fasta
    output: indelible_output/10/cleaned_aln/38.fasta, indelible_output/10/concatenated/38.fasta, indelible_output/10/cleaned_trees/38.nwk, indelible_output/10/cleaned_N0_trees/38.nwk, indelible_output/10/cleaned_no_internal_trees/38.nwk
    jobid: 2
    wildcards: taxon=10, rep=38

[Tue Mar  2 12:13:17 2021]
Error in rule clean_indelible:
    jobid: 2
    output: indelible_output/10/cleaned_aln/38.fasta, indelible_output/10/concatenated/38.fasta, indelible_output/10/cleaned_trees/38.nwk, indelible_output/10/cleaned_N0_trees/38.nwk, indelible_output/10/cleaned_no_internal_trees/38.nwk

RuleException:
CalledProcessError in line 51 of /Users/gabefoley/Dropbox/PhD/20210121_Indel_Evaluation_Project/Indel_Evaluation_Snakemake/Snakefile:
Command 'set -euo pipefail;  /Users/gabefoley/miniconda3_latest/envs/indel_snakemake/bin/python3.9 /Users/gabefoley/Dropbox/PhD/20210121_Indel_Evaluation_Project/Indel_Evaluation_Snakemake/.snakemake/scripts/tmp1l1bglon.clean_indelible.py' returned non-zero exit status 1.
  File "/Users/gabefoley/miniconda3_latest/envs/indel_snakemake/lib/python3.9/site-packages/snakemake/executors/__init__.py", line 2319, in run_wrapper
  File "/Users/gabefoley/Dropbox/PhD/20210121_Indel_Evaluation_Project/Indel_Evaluation_Snakemake/Snakefile", line 51, in __rule_clean_indelible
  File "/Users/gabefoley/miniconda3_latest/envs/indel_snakemake/lib/python3.9/site-packages/snakemake/executors/__init__.py", line 568, in _callback
  File "/Users/gabefoley/miniconda3_latest/envs/indel_snakemake/lib/python3.9/concurrent/futures/thread.py", line 52, in run
  File "/Users/gabefoley/miniconda3_latest/envs/indel_snakemake/lib/python3.9/site-packages/snakemake/executors/__init__.py", line 554, in cached_or_run
  File "/Users/gabefoley/miniconda3_latest/envs/indel_snakemake/lib/python3.9/site-packages/snakemake/executors/__init__.py", line 2350, in run_wrapper
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: /Users/gabefoley/Dropbox/PhD/20210121_Indel_Evaluation_Project/Indel_Evaluation_Snakemake/.snakemake/log/2021-03-02T121312.744449.snakemake.log
